docs/developer.md

```diff
@@ -1,57 +1,113 @@
-## Developer Guide
+# Developer Guide
```

Comment:
I'm a little confused - why is this file showing up in the diff? I'm ont seeing any changes from what's on main?